seminars/01-Naming/README.md

```diff
@@ -0,0 +1,16 @@
+# Names
```

Comment:
Is there a more fun/striking title? We know how important naming is, but to someone less familiar, _names_ just don't' feel that exciting.

Something like **THE POWER OF NAMING** might feel like overkill, but I feel like it's more fun?